Modules/@babylonjs/react-native/android/include/XrContextHelperARCore.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#if __has_include("jsi/jsi.h")
+#include "jsi/jsi.h"
+#include "IXrContextARCore.h"
+
+namespace BabylonReactNative

```

Comment:
It might make sense to follow the namespace naming scheme of Babylon Native here. So something like Babylon::Plugins::NativeXr as a clear indication that this is tied to that plugin.

Reply:
this should now be updated